spec/controllers/queue_items_controller_spec.rb

```diff
@@ -112,4 +111,80 @@
       expect(response).to redirect_to log_in_path
     end
   end
- end
+
+  describe "POST update" do
+    context "with valid inputs" do
+      let(:monk) { Fabricate(:video) }
+      let(:item_1) do
+        Fabricate(:queue_item, video: monk, user: current_user, list_order: 1)
+      end
+      let(:item_2) do
+        Fabricate(:queue_item, video: monk, user: current_user, list_order: 2)
+      end
+      let(:item_3) do
+        Fabricate(:queue_item, video: monk, user: current_user, list_order: 3)
+      end
+
+      before do
+        session[:user_id] = current_user.id
+        item_1
+        item_2
+        item_3
+      end
+
+      it "redirects to the my queue page if authenticated" do
+        post :update_queue,
+             queue_items_data: {:list_orders => [], :ratings => []}
```

Comment:
Why do you use old ruby hash syntax mixed with new ? don't. stick to one style - latest would be the best.